libcloud/common/base.py

```diff
@@ -270,7 +270,11 @@ def parse_body(self):
             return self.body
 
         try:
-            body = ET.XML(self.body)
+            try:
+                body = ET.XML(self.body)
+            except ValueError:
```

Comment:
I would prefer, if it's possible to try to catch a more specific exception (and maybe also check for an error message if nothing else works), but I'm fine with that for now.